oscarlab/graphene
# [meson] Compile Pal/Linux

## Description of the changes 

This moves compiling direct PAL's `libpal.so` from Makefiles into meson.

Makefile.am is intentionally left alone, because it is needed to compile `.lib/` (`Pal/lib`), which is not in scope for this PR.

This PR depends on #2351.
See also: #2289.

## How to test this PR? 

Build should succeed, tests should pass.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2376)
